lib/arjdbc/oracle/adapter.rb

```diff
@@ -0,0 +1,863 @@
+# NOTE: file contains code adapted from **oracle-enhanced** adapter, license follows
```

Comment:
@kares I think we decided to eliminate support for Oracle adapter since another gem supports Oracle for JRuby ruby-oci?